data/db/rf-leica.xml

```diff
@@ -11,15 +11,50 @@
     </mount>
 
     <camera>
-        <maker>Leica</maker>
+        <maker>Leica Camera AG</maker>
```

Comment:
Should probably keep <maker lang="en">Leica</maker> as well - both libraw and rawspeed sanitize the maker to just "Leica" (for RWLs mostly), which could be passed to lensfun.

Reply:
You're probably right that it should be safe to stick w/ the original Exif reported value - darktable passes that one indeed for lensfun lookup, and not the rawspeed sanitized short name used elsewhere in the UI, so I was maybe too quick to comment. ;) Probably fine to proceed like this, and wait for any regression reports... (hopefully none!)